### PR TITLE
7128-Medic rebrand

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

This PR: 
- Updates the logo and its spacing in the login page.
- Updates the favico and page title.
- Fixes minor css spacing.
- Updates dev@medicmobile.org and hello@medicmobile.org to @medic.org domain.
- Update "Medic Mobile" to "Medic" in strings, documentation and comments, however legal documents remains with legal name (copyright and licenses)

Note: Colours won't be updated as they require an UX study, font's are already in Noto-sans and title won't change for now.

Ticket: https://github.com/medic/cht-core/issues/7128
CHT-Android: https://github.com/medic/cht-android/pull/195

**Screenshots of new logo:** 
**1** Logo updated.

**2** Logo, favico and page title already updated.

**Screenshots of css fixes:**

**1** Spacing and dropdown cutting the selected item text
Screen Shot 2021-07-19 at 5 05 12 pm

**2** Circle with checkboxes and numbers display is off.
Screen Shot 2021-07-19 at 4 50 57 pm

**3** Circle with checkboxes and numbers display is off.
Screen Shot 2021-07-07 at 12 35 08 am

**4** Filter spacing uneven.
Screen Shot 2021-07-07 at 12 30 04 am

**5** Spacing between elements.
Screen Shot 2021-07-07 at 12 24 04 am

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @njogz, can you please have a look :) Thanks!

File: api/LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2013 Medic Mobile, 501(c)(3)  <hello@medicmobile.org>
+Copyright 2013-2021 Medic Mobile, Inc 501(c)(3) <hello@medic.org>
```
Comment (njogz) [inline api/LICENSE:1]: should this still read Medic Mobile?

**Review by njogz — CHANGES_REQUESTED** [review]
Great job on this. Just one question about a section where you still have Medic Mobile.

File: api/LICENSE (hunk 1 above)
Comment (latin-panda) [reply api/LICENSE:1]: Yes, copyrights or licenses (legal-ish) needs to stay as "Medic Mobile, Inc".

**Review by njogz — APPROVED** [review]

File: api/LICENSE (hunk 1 above)
Comment (njogz) [reply api/LICENSE:1]: Okay.

**Pull request merged**